### PR TITLE
Add compat data for background(-.*)? properties

### PR DESCRIPTION
**Pull request by teoli2003**

### CODE REVIEW

File: css/background-clip.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        "obsolete": false
+      }
+    },
+    "text": {
+      "Android": {
+        "support": null
+      },
+      "Chrome": {
+        "support": true,
+        "notes": ["Support it on the prefixed version of the property only; according https://webkit.org/blog/164/background-clip-text/ WebKit does not include text decorations or shadows in the clipping."]
```
Comment (Elchi3) [inline css/background-clip.json]: - Remove "it". 
- Use an HTML link.

File: css/background-position-x.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        "support": null
+      },
+      "status": {
+        "experimental": false,
+        "standardized": true,
+        "stable": true,
+        "obsolete": false
+      }
+    },
+    "Two-value syntax (support for offsets from any edit)": {
```
Comment (Elchi3) [inline css/background-position-x.json]: s/edit/edge

File: css/background-position-y.json [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        "support": null
+      },
+      "status": {
+        "experimental": false,
+        "standardized": true,
+        "stable": true,
+        "obsolete": false
+      }
+    },
+    "Two-value syntax (support for offsets from any edit)": {
```
Comment (Elchi3) [inline css/background-position-y.json]: s/edit/edge

File: css/background-position.json [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        "support": null
+      },
+      "status": {
+        "experimental": false,
+        "standardized": true,
+        "stable": true,
+        "obsolete": false
+      }
+    },
+    "Four-value syntax (support for offsets from any edit)": {
```
Comment (Elchi3) [inline css/background-position.json]: s/edit/edge

File: css/background-position.json [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+      "Android": {
+        "support": true
+      },
+      "Chrome": {
+        "support": "25.0"
+      },
+      "Chrome for Android": {
+        "support": true,
+        "notes": ["Some version support only experimental gradients prefixed with -webkit"]
```
Comment (Elchi3) [inline css/background-position.json]: Gradients? This is not in the original page

File: css/background-image.json [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+        "obsolete": false
+      }
+    },
+    "Gradients": {
+      "Android": {
+        "support": true
+      },
+      "Chrome": {
+        "support": "1.0",
+        "notes": ["Some version support only experimental gradients prefixed with -webkit"]
```
Comment (Elchi3) [inline css/background-image.json]: "Some versions only support ..." here and a few times below

File: css/background-repeast.json [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+      },
+      "Edge": {
+        "support": "12"
+      },
+      "Edge Mobile": {
+        "support": null
+      },
+      "Firefox": {
+        "support": "13.0",
+        "notes": ["Implemented in bug 548375"]
```
Comment (Elchi3) [inline css/background-repeast.json]: Why is this note needed?

File: css/background-repeast.json [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+      },
+      "Edge": {
+        "support": "12"
+      },
+      "Edge Mobile": {
+        "support": null
+      },
+      "Firefox": {
+        "support": "49.0",
+        "notes": ["Implemented in bug 548372"]
```
Comment (Elchi3) [inline css/background-repeast.json]: Why is this note needed?

File: css/background.json [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+        "support": null
+      },
+      "status": {
+        "experimental": false,
+        "standardized": true,
+        "stable": true,
+        "obsolete": false
+      }
+    },
+    "Support for background-size": {
```
Comment (Elchi3) [inline css/background.json]: just "background-size"

File: css/background.json [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+        "support": null
+      },
+      "status": {
+        "experimental": false,
+        "standardized": true,
+        "stable": true,
+        "obsolete": false
+      }
+    },
+    "Support for background-origin and background-clip": {
```
Comment (Elchi3) [inline css/background.json]: Make this two entries, "background-origin" and "background-clip"

**Comment by teoli2003** [comment]
All comments addressed.
I also noticed a wrong filename (fixed) and added some <code> in notes as we settled to use html there.

I rebased the patches and the commits to have a clean history :-)

Travis-ci is happy too.

**Comment by teoli2003** [comment]
Ok Florian and I have settled to use css/properties/xyz.json instead of css/xyz.json. So I will modify this PR to reflect this :-)

**Comment by teoli2003** [comment]
Moved to css/properties and rebased. Ready for review agian.

**Comment by Elchi3** [comment]
Great work \o/

**Pull request merged**